### ISSUE TITLE
google directions service status

### ISSUE THREAD

**Issue by paulvanderheijden90** [mrdis/MMM-google-route#11]
hello,

i have a failure of the "over_query_limit". it was working for a couple of days.

And is google charging me for the direction information??

<img width="205" alt="schermafbeelding 2019-01-21 om 12 30 35" src="https://user-images.githubusercontent.com/40613554/51471943-620fe280-1d78-11e9-8654-3c16c84bcdea.png">

**Comment by paulvanderheijden90** [comment]
Hello.
Is it posible to update during the day Every hours but in the morning every 5 minuts. Is that posible ?

Verstuurd vanaf mijn iPhone

**Comment by mrdis** [comment]
Currently it's not possible, but I may implement something to work in conjunction with [MMM-ModuleScheduler](https://github.com/ianperrin/MMM-ModuleScheduler) for such scheduling, which may be useful actually. I may work on it on the next days.

**Comment by paulvanderheijden90** [comment]
Ok Nice! Let me know

Verstuurd vanaf mijn iPhone

**Comment by stanvv** [comment]
Yes, this would be a very useful feature. 
The module works well now, I'm only facing two issues every now and then:
1) OVER_QUERY_LIMIT: about once a week, which is weird cause the refreshPeriod has been the same for last month (so I'd expect to receive it every day, or never). A time-frame specific update would be useful, like implemented at https://github.com/jclarke0000/MMM-MyCommute 
2)  UNKNOWN_ERROR: about once every 2 weeks, no idea what is causing this.

**Comment by mrdis** [comment]
I just pushed this new functionality, you can now use the scheduler to configure custom update schedules.
Check out [the readme](https://github.com/mrdis/MMM-google-route#using-mmm-modulescheduler-to-configure-the-route-refresh-interval)

**Comment by paulvanderheijden90** [comment]
Hello,

I add this to the config and installed everything like the mmm scheduler is telling me.

I put this in:

{
        		module: 'MMM-ModuleScheduler',
			config: {
			notification_schedule: [
                		// Refresh the route every minute from 7:00 AM to 8:00 AM, monday to friday
                		{ notification: 'MMM-google-route/refresh', schedule: '* 7 * * 1-5' }
            			]
			}
    		},


What do you meen with Just set "MMM-google-route/refresh" as the notification id in the notification_schedule configuration parameter <https://github.com/ianperrin/MMM-ModuleScheduler#scheduling-notifications> of the module, e.g.??

Can I set the refresh rate to 30 sec ?



Met vriendelijke groet,


Paul van der Heijden

**Comment by mrdis** [comment]
The configuration is ok, i meant the "notification" field of the object in the "notification_schedule" configuration parameter.
The refresh rate anyway cannot be set to less than one minute, that's a limitation of the scheduler module.

**Comment by stanvv** [comment]
Thanks for the hard word @mrdis , scheduler works well!

I'm still receiving the OVER_QUERY_LIMIT though, here is some more info:
- I've implemented the scheduler around 30th of January (see dotted line attached)
- Since then I've received 2x an OVER_QUERY_LIMIT , which [results in error 403](https://developers.google.com/maps/premium/previous-licenses/articles/usage-limits): February 2nd and yesterday
- They can basically be caused by either too many requests per day, or sending requests too fast as far as I know. According to the table attached I've sent about 5000 requests in 30 days, and I used your scheduling settings (every minute between 7 and 8, so 60 requests/day), so that should not be the issue. (FYI: refreshPeriod = 8) 
- I don't have any specific usage caps set in my project Quotas

So from that point of view if should be caused by sending requests too fast, for which [this test script](https://developers.google.com/maps/premium/previous-licenses/articles/usage-limits#limitexceeded) is created. I also found [this section below](https://developers.google.com/maps/premium/previous-licenses/articles/usage-limits#http403), which tells me the 403 error can also be caused by an invalid URL signature. But I have no idea how to continue from here.

Any thoughts? @paulvanderheijden90 , are you still receiving the OVER_QUERY_LIMIT error?
